person.rb

```diff
@@ -0,0 +1,17 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super()
```

Comment:
- [ ] There is no need to call the super method here since Person is not a child class of another one., (There is no inheritance). Let's remove it.